packages/contract/test/deploy.js

```diff
@@ -225,8 +224,8 @@ describe("Deployments", function() {
 
       assert(multiplier === 1.25, "Multiplier should be initialized to 1.25");
       assert(
-        multiplier * estimate > block.gasLimit,
-        "Multiplied estimate should be too high"
+        multiplier * estimate < block.gasLimit,
```

Comment:
In theory, this shouldn't be necessary anymore especially for tests; previously ganache would underestimate in rare cases (conditional branching on gasLeft).  Obviously, in reality, deploying runs the risk of state changes during the interim between estimates and transaction dispatch/propagation/block inclusion etc. and so a multiplier makes sense on any public networks.

Reply:
This is a very weird change for an upgrade PR. It looks like the old version was just semantically incorrect?

Reply:
@gnidan see prior convo here: https://github.com/trufflesuite/truffle/pull/2758#discussion_r366688384

Reply:
Thanks for the context @CruzMolina!

I did some digging to figure out the [original intention](https://github.com/trufflesuite/truffle/commit/5a06ecfd7b0b50dabbdca68dcf5d20812915f44c) of the test, and I think it's this:
- Create a situation where the default multiplier results in a value that exceeds the block limit
- Proceed to deploy anyway and not have an exception

If this reasoning is correct, then I think to fix this test, we'd need to update the `Example` contract or the default multiplier so that:
- estimate < block limit
- estimate * default multiplier > block limit

i.e., we want to test that multiplier is self-capping

Reply:
Re: your point @nicholasjpaterno 
> In theory, this shouldn't be necessary anymore especially for tests

We still need the multiplier because we still want to address cases where there's a state change in between the estimate call and the tx. (Although with your gas exactimation work, we may be able to change the default from 1.25 to something lower)